src/data/multidim/multidimindex.h

```diff
@@ -20,14 +20,14 @@ static inline std::string to_string(const MultiIndex &multiIndex)
 {
 	typedef Text::SmartString S;
 	return "[ "
-	     + S::join<std::vector, std::string>(
+	     + S::join(
```

Comment:
This function should be toJSON() as well. I missed this when I renamed the other ones.

Reply:
I'll change it in https://github.com/vizzuhq/vizzu-lib/pull/297 PR

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/297/commits/30d727829e141581170bc03d38d1c5cdc352678c